src/lasp_sup.erl

```diff
@@ -123,7 +123,7 @@ init(_Args) ->
     end,
 
     %% Configure Plumtree logging.
-    TransLogMFA = {lasp_instrumentation, transmission, [broadcast]},
+    TransLogMFA = {lasp_instrumentation, transmission},
```

Comment:
I think this, since it's a MFA, should be lasp_instrumentation, transmission, [] to make it at least match the module, function, arguments signature.